package.json

```diff
@@ -24,6 +24,7 @@
         "lodash": "4.17.4",
         "npm": "3.10.10",
         "opn": "4.0.2",
+        "precommit": "^1.2.2",
```

Comment:
Is there any particular reason, this node module is added.

Reply:
It was added when I was debugging an error preventing me from making PRs, but is not needed, so I've removed it.